cmd/move_config.go

```diff
@@ -66,12 +69,14 @@ func Move(dryRun bool) error {
 
 	log.Println("WARNING: Helm v3 configuration may be overwritten during this operation.")
 	log.Println()
-	doCleanup, err := utils.AskConfirmation("Move Config", "move the v2 configuration")
-	if err != nil {
-		return err
+	if skipConfirmation {
+		log.Println("Skipping confirmation before performing move.")
+		doCleanup = true
+	} else {
+		doCleanup, err = utils.AskConfirmation("Move Config", "move the v2 configuration")
```

Comment:
The check for `err != nil` is missing here.

Reply:
thanks added